lib/Net/DRI/Protocol/EPP/Extensions/Nomulus.pm

```diff
@@ -0,0 +1,77 @@
+## Domain Registry Interface, Nomulus (Google) extensions
```

Comment:
Should we link CRR and Nomulus modules? I think they are the same thing?

Reply:
When I merged all versions I simply kept them seperate just avoid possible problems and/or lose functionalities.

- https://github.com/paullojorgge/Net-DRI/tree/pm-Net-DRI-0.96_13/lib/Net/DRI/Protocol/EPP/Extensions/Nomulus

But definitely something that we need to review as soon as all versions are merged :)